service/src/main/java/bio/terra/workspace/service/resource/referenced/ReferencedGcsObjectAttributes.java

```diff
@@ -3,22 +3,23 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
-public class ReferencedGcsBucketFileAttributes {
+public class ReferencedGcsObjectAttributes {
   private final String bucketName;
-  private final String fileName;
+  private final String objectName;
 
   @JsonCreator
-  public ReferencedGcsBucketFileAttributes(
-      @JsonProperty("bucketName") String bucketName, @JsonProperty("fileName") String fileName) {
+  public ReferencedGcsObjectAttributes(
+      @JsonProperty("bucketName") String bucketName,
+      @JsonProperty("objectName") String objectName) {
```

Comment:
I think changing this field from `fileName` to `objectName` will break deserialization of any GCS object references which already exist in WSM's DB. 
That might be ok since you added these pretty recently (#497), but it's worth checking existing environments.

Reply:
there is no calling of this endpoint yet. so i think it should be OK?